packages/lucide-preact/tests/lucide-preact.spec.tsx

```diff
@@ -1,71 +1,51 @@
 import { describe, it, expect } from 'vitest';
-import { render, cleanup } from '@testing-library/preact'
+import { render, cleanup } from '@testing-library/preact';
 import { Pen, Edit2, Grid, Droplet } from '../src/lucide-preact';
 
-type AttributesAssertion = { attributes: Record<string, { value: string }>}
+type AttributesAssertion = { attributes: Record<string, { value: string }> };
 
 describe('Using lucide icon components', () => {
   it('should render an component', () => {
-    const { container } = render( <Grid/> );
+    const { container } = render(<Grid />);
 
-    expect( container.innerHTML ).toMatchSnapshot();
+    expect(container.innerHTML).toMatchSnapshot();
   });
 
   it('should adjust the size, stroke color and stroke width', () => {
     const testId = 'grid-icon';
     const { container, getByTestId } = render(
-      <Grid
-        data-testid={testId}
-        size={48}
-        stroke="red"
-        strokeWidth={4}
-      />,
+      <Grid data-testid={testId} size={48} stroke="red" strokeWidth={4} />,
```

Comment:
Can we enforce props on a new line? 
It's easier to read if props are on a new line.